test/namespace.spec.ts

```diff
@@ -0,0 +1,285 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import "mocha";
+import * as chai from "chai";
+const should = chai.should();
+import * as chaiAsPromised from "chai-as-promised";
+import * as dotenv from "dotenv";
+dotenv.config();
+chai.use(chaiAsPromised);
+import { Namespace } from "../lib";
+
+function testFalsyValues(testFn: Function) {
```

Comment:
Should this be called `testInvalidValues` or something that is more descriptive?